Assembler.Core/IMessageEnricher.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Assembler.Core.Entities;
+
+namespace Assembler.Core
+{
+    public interface IMessageEnricher<TFrame, TMessage>
```

Comment:
Wait, you have 2 generic types that you never use? Why is that?

Reply:
Oops, f

Reply:
👍